okhttp/src/main/kotlin/okhttp3/internal/cache/DiskLruCache.kt

```diff
@@ -242,6 +245,8 @@ class DiskLruCache(
 
     civilizedFileSystem = fileSystem.isCivilized(journalFileBackup)
 
+    cacheLock = openLock(fileSystem, directory)
```

Comment:
Is fileSystem ever an instance of `FileSystem.SYSTEM` I think we might wrap it ?